### PR TITLE
Added session-authentication-session-policies-global data source

### PR DESCRIPTION
**Pull request by PingDavidR**
- closes BRASS-1260

### CODE REVIEW

File: internal/resource/config/session/authenticationsessionpolicies/global/session_authentication_session_policies_global_data_source.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+}
+
+func readSessionAuthenticationSessionPoliciesGlobalDataSource(ctx context.Context, r *client.GlobalAuthenticationSessionPolicy, state *sessionAuthenticationSessionPoliciesGlobalDataSourceModel) {
+	state.Id = types.StringValue("session_authentication_session_policies_global_id")
+	state.EnableSessions = types.BoolValue(r.EnableSessions)
+	state.PersistentSessions = types.BoolPointerValue(r.PersistentSessions)
+	state.HashUniqueUserKeyAttribute = types.BoolPointerValue(r.HashUniqueUserKeyAttribute)
+	state.IdleTimeoutMins = types.Int64PointerValue(r.IdleTimeoutMins)
+	state.IdleTimeoutDisplayUnit = internaltypes.StringTypeOrNil(r.IdleTimeoutDisplayUnit, true)
```
Comment (henryrecker-pingidentity) [inline internal/resource/config/session/authenticationsessionpolicies/global/session_authentication_session_policies_global_data_source.go:112]: Do we need this method anymore? Can we just use types.StringPointerValue?

Comment (PingDavidR) [reply internal/resource/config/session/authenticationsessionpolicies/global/session_authentication_session_policies_global_data_source.go:112]: I think in a data source it makes sense to go with the framework default.  Changes made.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,17 @@
 * **New Resource:** `pingfederate_oauth_open_id_connect_policy` ([#105](https://github.com/pingidentity/terraform-provider-pingfederate/pull/105))
 
 #### Data Sources
+<<<<<<< HEAD
```
Comment (henryrecker-pingidentity) [inline CHANGELOG.md:7]: Committed merge conflicts here

Comment (PingDavidR) [reply CHANGELOG.md]: fixed

**Review by henryrecker-pingidentity — APPROVED** [review]

**Pull request merged**